plugins/source/gcp/resources/services/aiplatform/operations.go

```diff
@@ -30,6 +30,11 @@ func Operations() *schema.Table {
 				Resolver:   client.ResolveProject,
 				PrimaryKey: true,
 			},
+			// {
```

Comment:
Should we remove this comment?

Reply:
Removed in https://github.com/cloudquery/cloudquery/pull/15783/commits/de0a35bc5bd4fe949ffcd19f4bc9daeabd70a79c